internal/configs/ingress_test.go

```diff
@@ -161,6 +161,7 @@ func createExpectedConfigForCafeIngressEx() version1.IngressNginxConfig {
 						Upstream:            coffeeUpstream,
 						ProxyConnectTimeout: "60s",
 						ProxyReadTimeout:    "60s",
+						ProxySendTimeout:    "60s",
```

Comment:
all the changes in this file are not required. See the comment in the `internal/configs/version1/config.go` file.